inc/gui.class.php

```diff
@@ -160,74 +160,74 @@ public static function options_page() { ?>
-									<?php esc_html_e('BBCode is spam', 'antispam-bee') ?>
-									<span><?php esc_html_e('Review the comment contents for BBCode links', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'BBCode is spam', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Review the comment contents for BBCode links', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_advanced_check" id="ab_advanced_check" value="1" <?php checked($options['advanced_check'], 1) ?> />
 								<label for="ab_advanced_check">
-									<?php esc_html_e('Validate the ip address of commenters', 'antispam-bee') ?>
-									<span><?php esc_html_e('Validity check for used ip address', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Validate the ip address of commenters', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Validity check for used ip address', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_regexp_check" id="ab_regexp_check" value="1" <?php checked($options['regexp_check'], 1) ?> />
 								<label for="ab_regexp_check">
-									<?php esc_html_e('Use regular expressions', 'antispam-bee') ?>
-									<span><?php _e('Predefined and custom patterns by <a href="https://gist.github.com/4242142" target="_blank">plugin hook</a>', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Use regular expressions', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Predefined and custom patterns by <a href="https://gist.github.com/4242142" target="_blank" rel="noopener noreferrer">plugin hook</a>', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_spam_ip" id="ab_spam_ip" value="1" <?php checked($options['spam_ip'], 1) ?> />
 								<label for="ab_spam_ip">
-									<?php esc_html_e('Look in the local spam database', 'antispam-bee') ?>
-									<span><?php esc_html_e('Already marked as spam? Yes? No?', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Look in the local spam database', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Already marked as spam? Yes? No?', 'antispam-bee' ); ?></span>
 								</label>
 							</li>
 
 							<li>
 								<input type="checkbox" name="ab_dnsbl_check" id="ab_dnsbl_check" value="1" <?php checked($options['dnsbl_check'], 1) ?> />
 								<label for="ab_dnsbl_check">
-									<?php esc_html_e('Use a public antispam database', 'antispam-bee') ?>
-									<span><?php _e('Matching the ip address with <a href="http://www.stopforumspam.com/" target="_blank">Stopforumspam</a>', 'antispam-bee') ?></span>
+									<?php esc_html_e( 'Use a public antispam database', 'antispam-bee' ); ?>
+									<span><?php esc_html_e( 'Matching the ip address with <a href="http://www.stopforumspam.com/" target="_blank" rel="noopener noreferrer">Stopforumspam</a>', 'antispam-bee' ); ?></span>
```

Comment:
See above. I think, we can''t escape the translation as this would result in showing the escaped link and not showing a clickable link.